docs/faq/contributions.md

```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 6
+description: Earning Obol Contributions
+---
+
+# Obol Contributions Program
+
+## What are Contributions?
+
+The Obol Contributions Program acknowledges and rewards participants for their efforts in scaling and decentralizing the Ethereum network by staking on Distributed Validators (DVs). When you stake with DVs, 1% of the (consensus layer) staking rewards are directed to the Obol Collective’s “1% for Decentralization” retroactive funding program (RAF), and the portion of rewards you contribute to the RAF reflects your contributions.
+
+## How to Earn Contributions
+
+You can earn contributions by:
+
+1. **Running your own DV Cluster**: You can create and manage your own DV cluster using our Launchpad. This allows you to directly participate in the network's decentralization efforts and earn contributions. To get started, visit our [DV Launchpad](https://launchpad.obol.org). You can also visit our [Squad Staking](https://squadstaking.com) page for inspiration and to find cluster-mates.
+
+2. **Staking through Staking Partners**: By participating in staking through our partners, your staking rewards will also contribute to the Obol Collective. Here’s how you can start earning contributions through our partners:
+
+- **Lido / Mellow**: The Obol vault accepts ETH deposits into the Simple DVT Module, bridging robust institutional staking and decentralized native staking. [Learn more](https://lido.fi).
+- **Chorus One / Stakewise**: Stakewise and Chorus One offer a platform for ETH staking with integrated Obol DVs. [Learn more](https://opus.chorus.one/pool/stake).
+- **Stakely**: Our partner enabling contributions through DV staking. [Learn more](https://stakely.io/).
+
+## How Contributions are Accumulated
+
+Contributions are accumulated based on the staking rewards generated by your validators (pubkeys). Here's a quick overview of the process:
+
+1. **Staking Rewards**: Each DV contributes 1% of its (consensus layer) staking rewards to the Obol Collective. This means the more effective and uptime your validators have, the more rewards (and thus contributions) you will generate.
```

Comment:
I think we should be more descriptive here on how DV contributes 1% using splits and how validator rewards gets converted to operator contributions

i.e when clusters deploy splits with 1 % to  retroactivefunding.obol.eth,  validator CL rewards are rolled up into clusters with  and then split using split percentages to operators. 

You can also include the formulas with summations etc to make it easier to understand.

We should also mention contributions are updated once a day using of this formula.

We should include that Launchpad only shows direct contributions by Squads and Clusters. Users who contributed through partners will have to go through their UI to find contributions